beacon-chain/slasher/detect_attestations.go

```diff
@@ -193,15 +193,31 @@ func (s *Service) checkDoubleVotes(
 			// This is a double vote.
 			doubleVotesTotal.Inc()
 
-			existing, ok := existingAttWrapper.IndexedAttestation.(*ethpb.IndexedAttestation)
+			existingWrapper, ok := existingAttWrapper.IndexedAttestation.(*slashertypes.WrappedIndexedAtt)
+			if !ok {
+				return nil, fmt.Errorf(
+					"existing attestation has wrong type (expected %T, got %T)",
```

Comment:
Just to make it easier to identify the failure since otherwise these errors are exactly the same other than the types
```suggestion
					"existing attestation wrapper has wrong type (expected %T, got %T)",
```